### PR TITLE
Updates Hub Information as per Council Vote 1052

### PR DESCRIPTION
**Pull request by JamieD1**
:cl:  
tweak: Changes Hub Text
/:cl:

### CODE REVIEW

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
Revert the change from adventure to fun, this is not part of the council vote and will require another one.

**Comment by JamieD1** [comment]
Done your Majesty

File: yogstation/code/game/world.dm [hunk 1]
```diff
@@ -41,7 +41,7 @@ GLOBAL_LIST_EMPTY(donators)
 	var/server_name = CONFIG_GET(string/servername)
 	if (server_name)
 		s += "[server_name]\] &#8212; Dive in Now: Perfect for Beginners!"
-	s += "<br>99% Lag-Free <b>MRP</b> Experience! Join the Adventure!"
+	s += "<br>99% Lag-Free <b>Regular Events!!</b> Join the Adventure!"
```
Comment (Bibby0110) [inline yogstation/code/game/world.dm]: The council vote specified only 1 !, the second ! will require another vote

**Review by Bibby0110 — DISMISSED** [review]

File: yogstation/code/game/world.dm (hunk 1 above)
Comment (ToasterBiome) [inline yogstation/code/game/world.dm]:
```suggestion
	s += "<br>99% Lag-Free <b>Regular Events!</b> Experience! Join the Adventure!"
```

The council vote states Remove the MRP tag on BYOND, and instead replace it with "Regular Events!". This change removes the phrase "MRP" and replaces it with "Regular Events!". Removal of other words requires a separate council vote.

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
The council votes

**Comment by Hislyna** [comment]
Considering that we stated the MRP tag, and experiance being apart of that tag, experiance is to be removed.

**Comment by ToasterBiome** [comment]
> Considering that we stated the MRP tag, and experience being apart of that tag, experience is to be removed.

The vote doesn't state that.

**Comment by Hislyna** [comment]
> > Considering that we stated the MRP tag, and experience being apart of that tag, experience is to be removed.
> 
> The vote doesn't state that.

Doesn't need to, its apart of the tag. It did not state remove MRP, it stated remove the MRP tag.

**Comment by ToasterBiome** [comment]
> > > Considering that we stated the MRP tag, and experience being apart of that tag, experience is to be removed.
> > 
> > 
> > The vote doesn't state that.
> 
> Doesn't need to, its apart of the tag. It did not state remove MRP, it stated remove the MRP tag.

There is no scoping for this clarification. The MRP tag is "MRP". That's the entire tag. Unless you're also implying `99% Lag-Free` is also part of the tag?

**Comment by Hislyna** [comment]
> > > > Considering that we stated the MRP tag, and experience being apart of that tag, experience is to be removed.
> > > 
> > > 
> > > The vote doesn't state that.
> > 
> > 
> > Doesn't need to, its apart of the tag. It did not state remove MRP, it stated remove the MRP tag.
> 
> There is no scoping for this clarification. The MRP tag is "MRP". That's the entire tag. Unless you're also implying `99% Lag-Free` is also part of the tag?

Don't bring the 99% Lag-Free into this. Thats our main selling point of free lag!

**Comment by ToasterBiome** [comment]
> > > > > Considering that we stated the MRP tag, and experience being apart of that tag, experience is to be removed.
> > > > 
> > > > 
> > > > The vote doesn't state that.
> > > 
> > > 
> > > Doesn't need to, its apart of the tag. It did not state remove MRP, it stated remove the MRP tag.
> > 
> > 
> > There is no scoping for this clarification. The MRP tag is "MRP". That's the entire tag. Unless you're also implying `99% Lag-Free` is also part of the tag?
> 
> Don't bring the 99% Lag-Free into this. Thats our main selling point of free lag!

Come back with a warrant then.

**Comment by neeshacark** [comment]
this is the most petty thing to get caught up over guys please this is why people make fun of us

**Review by ToasterBiome — APPROVED** [review]
looks like shit mate

**Comment by wonderinghost** [comment]
> > > > > Considering that we stated the MRP tag, and experience being apart of that tag, experience is to be removed.
> > > > 
> > > > 
> > > > The vote doesn't state that.
> > > 
> > > 
> > > Doesn't need to, its apart of the tag. It did not state remove MRP, it stated remove the MRP tag.
> > 
> > 
> > There is no scoping for this clarification. The MRP tag is "MRP". That's the entire tag. Unless you're also implying `99% Lag-Free` is also part of the tag?
> 
> Don't bring the 99% Lag-Free into this. Thats our main selling point of free lag!

relatively, however with higher pops you get problems of constant reconnection issues. sometimes stacked after the moment of reconnection or during the game

**Pull request merged**